packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -542,35 +543,70 @@ class EditorPage {
 		return await blockLocator.text();
 	}
 
+	async getNumberOfParagraphBlocks() {
+		const paragraphBlockLocator = isAndroid()
+			? `//android.view.ViewGroup[contains(@content-desc, "Paragraph Block. Row")]/android.widget.EditText`
+			: `(//XCUIElementTypeButton[contains(@name, "Paragraph Block. Row")])`;
+		const locator = await this.driver.elementsByXPath(
+			paragraphBlockLocator
+		);
+		return locator.length;
+	}
+
+	async assertSlashInserterPresent() {
+		const slashInserterLocator = isAndroid()
+			? '//android.widget.HorizontalScrollView[@content-desc="Slash inserter results"]/android.view.ViewGroup'
+			: '(//XCUIElementTypeOther[@name="Slash inserter results"])[1]';
+
+		return await isElementVisible( this.driver, slashInserterLocator );
+	}
+
 	// =========================
 	// List Block functions
 	// =========================
 
-	async getTextViewForListBlock( block ) {
-		let textViewElementName = 'XCUIElementTypeTextView';
-		if ( isAndroid() ) {
-			textViewElementName = 'android.widget.EditText';
-		}
+	async getListBlockAtPosition(
+		position = 1,
+		options = { isEmptyBlock: false }
+	) {
+		// Go to the correct list at position
+		const listBlockPositionLocator = isAndroid()
+			? `//android.view.ViewGroup[@content-desc="List Block. Row ${ position }"]`
+			: `(//XCUIElementTypeOther[contains(@name, "List Block. Row ${ position }")])[5]`;
```

Comment:
Whoops, my bad, I was testing with the number I got from Appium inspector and committed that 😬  I've updated this to 1 in the latest commit, the reason is that there multiple are returns and it needs to wait for at least 1 to be visible so 1 would be a safer choice rather than 5. 

Updated in this commit: https://github.com/WordPress/gutenberg/pull/40670/commits/d1af9076afc1a54787e003dd167aa1d17e0e1e21